./run_perfeval_2.sh: line 52: ./driver_perfeval_isa.py: No such file or directory
real 0.00
user 0.00
sys 0.00
Sat Apr 30 11:12:21 PDT 2016
Sat Apr 30 11:12:21 PDT 2016
